utilities/ubi8-asciidoctor/Dockerfile

```diff
@@ -21,37 +22,27 @@ ENV ASCIIDOCTOR_VERSION=${asciidoctor_version} \
   ASCIIDOCTOR_MATHEMATICAL_VERSION=${asciidoctor_mathematical_version} \
   ASCIIDOCTOR_REVEALJS_VERSION=${asciidoctor_revealjs_version} \
   KRAMDOWN_ASCIIDOC_VERSION=${kramdown_asciidoc_version} \
+  ASCIIDOCTOR_BIBTEX_VERSION=${asciidoctor_bibtex_version} \
   PANDOC_VERSION=${pandoc_version}
 
 
 USER root
 
-RUN dnf install -y \
-  tar \
-  gzip \
-  python3 \
-  python3-setuptools \
-  git \
-  ruby \
-  && dnf clean all && rm -rf /var/lib/dnf && rm -rf /usr/share/doc
-
-# install pandoc
-RUN curl -L https://github.com/jgm/pandoc/releases/download/${PANDOC_VERSION}/pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz | tar xvz --strip-components 1 -C /usr/local/
 
 # Installing Ruby Gems needed in the image
 # including asciidoctor itself
 # TODO: have to find Lasem dev libs     "asciidoctor-mathematical:${ASCIIDOCTOR_MATHEMATICAL_VERSION}" \
+# TODO: have to find a way to get graphviz and java working
```

Comment:
nit: maybe expand on what this comment means so someone else can pick up

Reply:
Lesson learned, do not add TODO as a reminder to myself for further maybe enhancements. Create a separate ticket for those enhancements.
Graphviz is not available in the default ubi8 repositories. So it has to be compiled from source. That compiling of source is the reason for the TODO.